src/front/app/features/profile-challenges/ProfileChallengeList.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component, PropTypes } from 'react';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+import Layout from '../Layout';
+import Menu from './ProfileChallengeMenu';
+import ChallengeTileList from '../../common/components/challenge/ChallengeTileList';
+
+class WatchList extends Component {
```

Comment:
Are you sure it is a good renaming? As per [requirements](https://tree.taiga.io/project/kucherenko-travel-and-help/us/5?milestone=77480) we have two lists of challenges: on the user screen and the watch list:

> - accepted challenges section with list of LAST 3 accepted challenges and 'show all' option - created challenges
> - section with list of LAST 3 created challenges and 'show all' option - completed challenges section with list of
> - LAST 3 completed challenges and 'show all' option.

Won't such renaming introduce additional confusion?